files/nrpe/check_openstack.py

```diff
@@ -672,6 +681,53 @@ def execute(self):
     except:
       raise
 
+class OSL3Agent():
+  '''
+  Check all Neutron L3 agents are alive
+  '''
+
+  options = dict()
+
+  def __init__(self, options):
+    self.neutron = neutronclient.Client('2', session=keystone_session_v3(options))
+
+  def check_bad_l3_agents(self):
+    err_l3agents = []
+    l3agents = self.neutron.list_agents(agent_type='L3 agent')
+
+    if str(l3agents.keys()) == "[u'agents']":
+      for item in l3agents.values():
+        for x in item:
```

Comment:
x is too short name for a variable, and doesn't give context of what is used for (also item doesn't give much context of the type of item). Also since the documentation on the python module for neutron is so little, it's difficult to find what is actually returned from the list_agents so this x and item became more obscure.

Reply:
Fixed